pyproject.toml

```
@@ -72,7 +72,7 @@ optional = true
 
 [tool.poetry.group.test.dependencies]
 pyasn1 = "*"
-pymongo = "^3"
+pymongo = ">=3.5"
```

I think we should do `<5` to be on the safe side
```suggestion
pymongo = ">=3.5, <5"
```

---

agreed